purchase_force_invoiced_quantity/views/purchase_order.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="priority" eval="99" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='order_line']/tree/field[@name='qty_invoiced']"
+                position="after"
+            >
+                <field
+                    name="force_invoiced"
+                    optional="show"
+                    attrs="{'column_invisible': [('parent.state', 'not in', ['purchase', 'done'])]}"
```

Comment:
we should set `readonly` attribute with the same condition despite it already being invisible

Reply:
ok, i will add this
but why do we need this?
could you explain, please?

Reply:
it's for not letting user modify the value in any case when it isn't confirmed or done

maybe considering not letting user edit field when it's done as well (meaning it's already locked)